ui/src/stylesheets/common.css

```diff
@@ -198,6 +198,16 @@ main {
     color: rgb(var(--v-theme-on-primary));
 }
 
+
+/* Make sure Button fills the containing group */
+.v-btn.v-btn--density-default  {
+    height: auto;
+}
+
+.v-btn--variant-elevated, .v-btn--variant-flat {
+    background-color: rgb(var(--v-theme-primary));
+    color: rgb(var(--v-theme-on-primary));
+}
```

Comment:
This was removed, so as part of your merge, it should've been removed too. When you're doing a Pull Request comparison like this, it will only ever show the difference between `main` and your branch, so this showing in green means it's being added as part of your work.

If we can get `line 206 - 210` removed, then this is good to go.